src/services/navigation/useLocationParams.ts

```diff
@@ -1,18 +1,23 @@
+import { History, Location } from 'history'
 import omitBy from 'lodash/omitBy'
 import qs from 'qs'
 import { useHistory, useLocation } from 'react-router-dom'
 
-export function useLocationParams(defaultParams = {}) {
-  const { push } = useHistory()
-  const { pathname, search, state } = useLocation()
+interface LocationParams {
+  [key: string]: string | number | boolean | any[]
```

Comment:
There may potentially be an object being set somewhere in the app, we may want to account for that